packages/analyticsdx-template-lint/src/schemas/template-info-schema.json

```diff
@@ -1207,6 +1122,50 @@
     }
   },
   "definitions": {
+    "condition": {
+      "type": ["null", "string"],
+      "defaultSnippets": [
+        {
+          "label": "\"\"",
+          "body": "${0}"
+        },
+        {
+          "label": "${...} expression",
+          "body": "${Variables.${1:varname}}"
+        }
+      ]
+    },
+    "onFailure": {
+      "type": ["null", "object"],
+      "description": "Controls how to handle failures when processing this asset. Only available in Winter '24 release.",
```

Comment:
@mblumreich This and the `enumDescriptions` below will show in hover text, let me know if you want updates.